### DISCUSSION TITLE
sumatrapdf error

### DISCUSSION THREAD

**Discussion by hueldoeu** [sumatrapdfreader/sumatrapdf#4074] [General]
opening the recently opened (pdf-)files with sumatrapdf, showing them in tabs is a nice thing but it happens often that i just want to take a look at a file, close sumatrapdf, delete the file and open sumatrapdf. then always this disturbing message appears. i don't want to see it anymore.
![Screenshot 2024-02-10 151256](https://github.com/sumatrapdfreader/sumatrapdf/assets/11273667/640e1731-2b2b-4cb2-bf1e-849464ccdddb)
please insert an option, that says if the user wants to see the message or not. thank you.
sincerely

**Comment by GitHubRulesOK** [comment]
If there were no error messages there would be no way to know there is a fault,

The message should disappear after a short delay

So the cause here appears, to be you are attempting to open a file tab that no longer has a file to open

It us due to closing a session with that tab still active then on session opening you are warned the session cannot find the file.

IF you know you will delete the file close the tab so it is not attempted on session restore.   

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/112ba2b0-5e21-4750-9acd-92278da8d110)

If it is always the task to open and delete then set restore to false

```
RestoreSession = false
```

**Comment by hueldoeu** [comment]
> The message should disappear after a short delay

this is a lie

> If it is always the task to open and delete then set restore to false
> 
> ```
> RestoreSession = false
> ```

and this i knew already but RestoreSessions isn't a bad idea. it's just the yellow message who has to be clicked every ******* time.

i am ok with it if the message goes away after scrolling few lines on the document with the mouse wheel.

**Comment by GitHubRulesOK** [comment]
The message should disappear after a short delay = It does for me using current pre-release (or first case 3.1.2 - 3.4.6)

3.5.2 should revert to home screen without the message AHH that may be your issue
**a single file not accessible on 3.5.2 will stay with that message active until another file chosen**

but in pre-release, the message disappears and tab stays active with the main reason !! so it has been addressed in a different fashion.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/515128a2-02ec-42bf-9ed2-8c700da6b674)
